.eslintrc.js

```diff
@@ -43,8 +53,6 @@ module.exports = {
         '@typescript-eslint/no-shadow': 'error',
         'no-unused-vars': 'off',
         '@typescript-eslint/no-unused-vars': ['error'],
-        'no-else-return': 'off',
-        'no-return-await': 'off'
```

Comment:
If we allow return awaits, it is easier to remember to use await if we later refactor the last line into a separate variable:
E.g. `return getFoo()` vs. `return await getFoo()`
refactored to
```
foo = await getFoo()
return foo
```

Reply:
Hm, not sure why you'd refactor it into a separate variable though? so it's named?

Either way it doesn't make any real difference other than introducing an additional microtask, right?